plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -5565,45 +5582,533 @@ img.help_tip {
+		}
+		#title {
+			border-radius: $border-radius;
+		}
+	}
+
+	// Gap between Title and Slug edit should be 8px = $gap-smaller, so we need to clear margins.
+	#poststuff #titlediv .inside {
+		padding: 0;
+		margin-top: $gap-smaller;
+	}
+	#edit-slug-box {
+		padding: 0;
+		margin: 0;
+	}
+	#titlediv #title {
+		margin: 0;
+	}
+
+	// Postbox cards.
+	.postbox {
+		border: 0;
+		border-radius: $border-radius;
+		display: flex;
+		flex-direction: column;
+	}
+	// Mimic wider gap between postboxes.
+	.post-body-content,
+	.postbox {
+		margin-bottom: 28px;
+	}
+	#postdivrich {
+		margin-top: 28px;
+		.wp-editor-wrap {
+			margin: $gap $gap 0;
+		}
+		#post-status-info {
+			margin: 0 $gap $gap;
+			width: calc(100% - 2 * $gap);
+		}
+	}
+
+	// Post/meta boxes.
+	.postbox-header {
+		padding: 12px 12px 12px 16px;
+		flex-shrink: 0;
+		align-self: stretch;
+		border-color: $separator-color;
+		min-height: 30px;
+
+		.hndle {
+			gap: $gap-small;
+			flex-wrap: wrap;
+		}
+	}
+	.closed .postbox-header {
+		border: 0;
+	}
+
+	.inside,
+	#commentsdiv.postbox .inside,
+	#postimagediv .inside {
+		padding: 16px;
+
+		// Remove margins of paragraph wrappers that probably should have been divs separated with gap.
+		p:first-child {
+			margin-top: 0;
+		}
+		p:last-child {
+			margin-bottom: 0;
+		}
+	}
+
+	// Separate rule, not to overspecify the above ones with id selector.
+	#poststuff {
+		h2.postbox-header{
+			padding: 12px 16px;
+		}
+		.hndle {
+			padding: 0;
+		}
+
+		.inside {
+			margin: 0;
+		}
+	}
+
+	.wp-editor-tools {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: space-between;
+	}
+
+	// Rich editor tabs.
+	.wp-editor-tabs {
+		display: flex;
+		align-self: flex-end;
+
+		.wp-switch-editor {
+			float: none;
+			background: none;
+			margin: 0;
+			padding: 10px $gap;
+			color: var(--Alias-text-text-tertiary, $alias-text-tertiary);
+			border-width: 0 0 2px 0;
+			border-bottom-color: transparent;
+			justify-content: center;
+			align-items: center;
+		}
+	}
+	.html-active .switch-html,
+	.tmce-active .switch-tmce {
+		border-bottom-color: var(--wp-admin-theme-color, $studio-wordpress-blue);
+		color: var(--Alias-text-text, $alias-text);
+	}
+
+
+	.wp-media-buttons {
+		display: flex;
+		align-self: flex-start;
+		flex-wrap: wrap;
+		gap: $gap-small;
+		padding-bottom: $gap;
+	}
+	.wp-editor-tools::after {
+		content: none;
+		display: none;
+	}
+
+	#postexcerpt > .postbox-header > .hndle > .woocommerce-help-tip,
+	#wp-content-media-buttons > .woocommerce-help-tip {
+		margin: 0;
+	}
+
+	// Product/coupon data options panel.
+	.woocommerce_options_panel {
+		.options_group:not(.hidden),
+		// General coupos data does not have options group wrapper, make it look like it has.
+		&#general_coupon_data:not(.hidden) {
+			display: flex;
+		}
+		.options_group,
+		&#general_coupon_data {
+			border-bottom: 1px solid #ededed;
+			padding: 36px 48px;
+			display: flex;
+			flex-direction: column;
+			gap: $gap-small;
```

Comment:
I'll address this one after extracting code to a separate file, as I plan to remove the `flex` layout and get back to the previous padding-based implementation. I hope it will be less disruptive for extensions contributing their content like before.

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/52632/files/7360212ecdd57857565780bbcd660e734e7773ad..504c312c1549aa3f852923b5f28cd9e3a9df062f